operator/pkg/lbipam/lbipam.go

```diff
@@ -604,22 +615,25 @@ func (ipam *LBIPAM) stripOrImportIngresses(sv *ServiceView) (statusModified bool
 				continue
 			}
 
-			err = lbRange.alloc.Alloc(ip, true)
+			serviceViews := make([]*ServiceView, 0, 1)
+			serviceViews = append(serviceViews, sv)
```

Comment:
Small nit, but you can write this faster

```suggestion
			serviceViews := []*ServiceView{sv}
```